code/datums/components/rad_vision.dm

```diff
@@ -0,0 +1,38 @@
+/datum/component/rad_vision
+
+/datum/component/rad_vision/Initialize()
+	START_PROCESSING(SSradiation, src)
```

Comment:
Use a different subsystem, SSobj is fine, SSradiation is for radiation spreading